### DISCUSSION TITLE
Meta Quest development.

### DISCUSSION THREAD

**Discussion by marcos-macedo** [korlibs/korge#2250] [Ideas and discussions]
Has anyone tried Meta Quest development with Korge? How difficulty would it be with adapting it to make it work?

As a concept, Korge looks perfect for VR game development.

**Comment by soywiz** [comment]
Hi Marcos. Nobody tried a s long as I'm concerned. 

I don't have any insights on how Meta Quest development works so I could not evaluate the difficulty on that

There are plans in for the views to be decouple from rendering. In the case Meta offers some kind of rendering.

Could I ask about the motivation on this topic?

**Reply by marcos-macedo** [reply]
As is known, Meta Quest is a VR/AR device. Its arquiteture is a linux sistem with an Android VM iver it. You can develop to it using both Native or Android. https://developer.oculus.com/quest/

You can also already develop for this platform using Unity or Unreal, but Korge and Kotlin would be much better. I think that AR/VR may be the future in gaming and it would be very interesting to find if someone is going this way with Korge.

I don´t really expect it to be out of the box, but it would be very interesting if we had someone exploring this way with Korge 3D and AR.

**Reply by soywiz** [reply]
I see. I would guess the way to go would be implementing OpenXR right?

**Reply by marcos-macedo** [reply]
Looks like it. That would be the way to get Korge into AR.

It would be amazing to have Meta listing Korge along side Unreal as a tool for developing to Meta Quest.

**Reply by soywiz** [reply]
Yeah. Would be awesome indeed. If someone wants to take care of it before the next roadmap update it would be great 👍
